### PR TITLE
feat: scheduled questions

### PR DESCRIPTION
**Pull request by tandpfun**

### CODE REVIEW

**Comment by mmmmmmassssonnn** [comment]
LGTM!

File: src/commands/scheduledQuestion.ts [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+      return ctx.reply(
+        `${ctx.client.EMOTES.checkmark} Created a ${frequency.toLowerCase()} scheduled question!`
+      );
+    } else if (ctx.args[0] === 'remove') {
+      await ctx.client.database.deleteScheduledQuestionChannel(ctx.channelId);
+
+      return ctx.reply(
+        `${ctx.client.EMOTES.checkmark} Removed the scheduled question in this channel.`
+      );
```
Comment (Benricheson101) [inline src/commands/scheduledQuestion.ts:113]: This doesn't check if a scheduled question exists, so you can run it without anything configured and the bot responds with success.


CleanShot 2022-12-16 at 13 19 54@2x

File: src/commands/scheduledQuestion.ts [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        embeds: [
+          {
+            title: `${ctx.client.EMOTES.time} Scheduled Question Channel List`,
+            color: ctx.client.COLORS.BLUE,
+            description: scheduledQuestionsForGuild
+              .map(
+                sc =>
+                  `• **${ctx.client.functions.titleCase(sc.schedule)}:** <#${sc.id}> - ${
+                    sc.type || ''
+                  } ${sc.rating || ''} ${sc.role ? `Pings <@&${sc.role}>` : ''}`
```
Comment (Benricheson101) [inline src/commands/scheduledQuestion.ts:133]: Rating is not displayed.

CleanShot 2022-12-16 at 13 33 20@2x

**Review by Benricheson101 — COMMENTED** [review]
Something else worth noting: there is no discrimination between the PG bot and R bot. Setting up scheduled questions with one bot and running the `view` command with the other will show the scheduled question as if it were set up with that bot.

File: src/commands/scheduledQuestion.ts (hunk 1 above)
Comment (mmmmmmassssonnn) [reply src/commands/scheduledQuestion.ts:113]: lol

**Pull request merged**